src/exe/Xournalpp.cpp

```diff
@@ -37,6 +43,36 @@ auto main(int argc, char* argv[]) -> int {
     _putenv_s("PANGOCAIRO_BACKEND", "fc");
 #endif
 
+#ifdef __APPLE__
+    /**
```

Comment:
To keep this file cleaner, I think it may be better to put the macOS initialization code into a `src/exe/osx` folder.